srcpkgs/vips/template

```diff
@@ -1,10 +1,14 @@
 # Template file for 'vips'
 pkgname=vips
-version=8.10.6
+version=8.11.0
 revision=1
 build_style=gnu-configure
 build_helper=gir
-configure_args="--enable-introspection=$(vopt_if gir yes no)"
+# TODO: As of version 8.11 vips supports loading its support for OpenSlide,
+# libheif, Poppler and libMagick backends at rundtime using GModule. Consider
+# tacking advantage of this feature and shipping these modules as optional
+# subpackages.
```

Comment:
Not sure how I feel about this; it would make installing a full vips harder (and would immediately degrade the experience for people who have `vips` installed, unless the core was shunted off to a `vips-core` package or similar). But I don't think the complexity is likely to be worth it.

Reply:
The `libMagick` dependency is currently not included because one of the major reasons to use vips over ImageMagick is that it is much smaller/cleaner/more auditable and more suited to use with untrusted image inputs.

From the vips readme's [ImageMagick section](https://github.com/libvips/libvips#imagemagick-or-optionally-graphicsmagick):
> If you are going to be using libvips with untrusted images, perhaps in a web server, for example, you should consider the security implications of enabling a package with such a large attack surface.

Reply:
Anyhow, I haven't looked into the other libraries they now support loading as modules and I'm not running vips on untrusted images myself so I don't have a very strong opnion on this.

This also isn't likely to make it to the top of my TODO list anytime soon.

Reply:
Can you make this clear in the comment, then? "The module approach would make enabling the imagemagick backend less of a compromise, it could maybe be the only one shunted off into its own subpackage" or somthing to that effect :)

Reply:
Done :+1: